suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -37,6 +37,7 @@ import {
 import { DiscoveryStatus } from '@suite-common/wallet-constants';
 import { requestDeviceAccess } from '@suite-native/device-mutex';
 import { analytics, EventType } from '@suite-native/analytics';
+import { isArrayMember } from '@trezor/type-utils';
```

Comment:
`type-utils` package is suposed to contain types only without runtime code.

Reply:
Done in #15019